Makefile

```diff
@@ -25,9 +25,16 @@ run-ios: .npminstall config/config.secret.json
 		echo "xcode is not installed"; \
 		exit 1; \
 	fi
+	@if ! [ $(shell command -v watchman) ]; then \
+		echo "watchman is not installed"; \
+		exit 1; \
+	fi
+
 	@echo Running iOS app in development
 
 	npm run run-ios
+	open -a Simulator
```

Comment:
Don't we also want to check for `watchman` in `run-android`? I'll add that real quick.